### DISCUSSION TITLE
How to filter "Action Items" tab ?

### DISCUSSION THREAD

**Discussion by immobilia** [apache/netbeans#3899] [General]
Same question as [NetBeans - lots of unnecessary Java warnings](https://stackoverflow.com/questions/71667515/netbeans-lots-of-unnecessary-java-warnings) on stackoverflow.

How to reduce default filter on "Action Items" tab ?
To exclude warnings : "This anonymous inner class creation can be turned into a lambda expression" 

I disable warnings :
![](https://i.stack.imgur.com/jPOgc.png)

![](https://i.stack.imgur.com/VfJVy.png)

But warnings stays listed on "Action Items" tab (even after clean/build, clean cache and NetBeans relaunch) :

![](https://i.stack.imgur.com/UAAFA.png)

The only change, are the underlined code warning, has gone :

![](https://i.stack.imgur.com/N0rYv.png)

**Comment by mbien** [comment]
sounds like a bug. I am pretty sure the window should aggregate the hints based on its filter settings (e.g display only hints of selected project etc). I am not using the Action Items view so I never noticed.

but in general:
you can configure the hints in two places, globally (tools->options->editor->hints) or per-project in the project preferences. If you disabled it so that its not showing in the editor it should also not show in the actions list IMO.

**Reply by immobilia** [reply]
I have no custom configuration in Project Properties :

![Project Properties](https://user-images.githubusercontent.com/438549/161848604-740ae450-d872-44ab-a5a3-c689eb1dfb0d.png)

"Edit IDE-wide options..." open the globals options, with options previously disabled as suggested in NetBeans suggest contextual dialog.

![image](https://user-images.githubusercontent.com/438549/161849075-7758fe45-70bd-4ad6-9553-06d2e79aded9.png)

I tried to disabling on one project, but there is not my options ("Suggestions" has few options, and not "Convert Anonymous to Member" nether "Convert Lamdba Expression ..."

![image](https://user-images.githubusercontent.com/438549/161850370-f0b12760-9cb0-4780-972d-dd580a2c013d.png)

**Reply by mbien** [reply]
i haven't forgotten about this. I just haven't had time yet to take a closer look.

> I tried to disabling on one project, but there is not my options ("Suggestions" has few options, and not "Convert Anonymous to Member" nether "Convert Lamdba Expression ..."

i think thats under "JDK Migration Support" -> "Convert Lambda or Member Reference", right?

**Reply by ghost** [reply]
It's been more than three years and this problem still persists. Unbelievable, Netbeans!
